scripts/release/docker-build-all

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+. ~/.nvm/nvm.sh
```

Comment:
This isn't very portable... you might want to check for the presence of `NVM_DIR`, and then error out if undetected?

Reply:
This doesn't need to be very portable as it's targetting the specific Docker image built in Dockerfile.  The issue is that `nvm` manipulates bashrc which doesn't seem to terminate early if it's non interactive session.  I can change it so bashrc is always honored, I'll attempt and follow up with a commit if I succeed.